libcob/ChangeLog

```diff
@@ -676,6 +681,11 @@
 	* fileio.c (cob_file_external_addr): change the way the file is allocated,
 	  having only its POINTER be "external" and use cob_file_malloc to allocate
 	  and set the attributes
+	* common.h (cob_module): new attribute gc_version
+	* common.c (cob_check_version): set last modules gc_version
+	* fileio.c (cob_cache_file): fix aborts when running GC 2 modules with
+	  INDEXED/RELATIVE files by converting old cob_file_key structure
+	  to the expected one
```

Comment:
I guess you just did not have to copy anything over here as there is no such aborting code/situation any more (leaving the 3.x changelog entries in is still useful)

Reply:
Yeah indeed, that was from 4392, which adds `gc_version` and that fix about `cob_cache_file`, which is no longer relevant (relies on the `flag` field from keys, which has been removed).

Actually I was wondering if I should keep those parts of changelogs when they refer to things that were removed in GC4 ; as this seems okay I'll keep them.